src/icp.cpp

```diff
@@ -142,7 +143,7 @@ usage() {
         "sockets getting timed out by the firewall as reported by some users.",
         " ",
         "Options are:",
-        " -Q  use RBUDP (datagram) protocol for the data transfer",
```

Comment:
There's an empty space to the left of `-Q`. Please remove it.

Reply:
And I think the space after will make it not align with the other lines beneath it...

Reply:
Agreed.

@JustinKyleJames Please align the description to match the others as well.

Reply:
I actually added a space in front of all of the dash options to match the formatting in iget, iput, and irepl.